private/src/avb_streamhandler/IasAlsaHwDeviceHandler.cpp

```diff
@@ -234,10 +234,17 @@ IasAvbProcessingResult IasAlsaHwDeviceHandler::init(uint16_t numChannels, uint32
        if (eIasAvbProcOK == ret)
        {
          IasLibPtpDaemon* mPtpProxy = IasAvbStreamHandlerEnvironment::getPtpProxy();
-         mOptimalFillLevel = optimalFillLevel;
-         mAlsaDeviceType   = alsaDeviceType;
-         mLastPtpEpoch     = mPtpProxy->getEpochCounter();
-         IasAvbStreamHandlerEnvironment::getConfigValue(IasRegKeys::cAudioTstampBuffer, mDescMode);
+         if (mPtpProxy)
```

Comment:
Minor nitpick: i think that in this project, they usually do the `x != nullptr` or even the `nullptr != x` (that I think it's ugly). But it's minor, you can merge without changing this.